SwrveSDK/src/main/res/values/themes.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="Theme.InAppMessageWithToolbar" parent="Theme.InAppMessage">
+        <item name="android:windowNoTitle">false</item>
+    </style>
+
+    <style name="Theme.InAppMessage" parent="android:Theme">
```

Comment:
I think the parent of Theme.InAppMessage needs to be @android:style/Theme.NoTitleBar.Fullscreen and parent of 
Theme.InAppMessageWithToolbar should be @android:style/Theme

And there should be the same styles in a values-v14 folder but with parents of :
@android:style/Theme.Holo.Light
@android:style/Theme.Holo.Light.NoActionBar.Fullscreen

If its done this way then the following line is not needed in SwrveInAppMessageActivity:
https://github.com/Swrve/swrve-android-sdk/pull/195/files#diff-68404ffda1fc09812527ef3d28582b28R83